bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/SummernoteEditor.html

```diff
@@ -0,0 +1,10 @@
+<html xmlns:wicket="http://wicket.apache.org">
+	<head>
+		<title>SummernoteEditor</title>
+	</head>
+	<body>
+		<wicket:panel>
+			<div id="summernote"></div>
```

Comment:
This hardcoded id looks problematic.
You cannot have more than one editor per page this way.